superset-frontend/src/visualizations/TimeTable/SparklineCell.tsx

```diff
@@ -18,39 +18,36 @@
  * under the License.
  */
 import React from 'react';
+import moment from 'moment';
 import {
-  Sparkline,
+  formatNumber,
+  formatTime,
+  getTextDimension,
+  useTheme,
+} from '@superset-ui/core';
+import { GridRows } from '@visx/grid';
+import { LinearScaleConfig, scaleLinear } from '@visx/scale';
+import { AxisScaleOutput } from '@visx/axis';
+import {
+  Axis,
   LineSeries,
-  PointSeries,
-  HorizontalReferenceLine,
-  VerticalReferenceLine,
-  WithTooltip,
-} from '@data-ui/sparkline';
-import { getTextDimension, formatNumber } from '@superset-ui/core';
+  Tooltip,
+  XYChart,
+  buildChartTheme,
+} from '@visx/xychart';
 
 interface Props {
-  ariaLabel: string;
```

Comment:
Could we instead keep using it and pass it as the `accessibilityLabel` of the `XYChart`? https://airbnb.io/visx/docs/xychart#XYChart_accessibilityLabel